docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,261 @@
+.. _ledger-nano:
+
+Manage XNS via Ledger Nano hardware wallet
+==========================================
+
+`Ledger Nano S <https://shop.ledger.com/products/ledger-nano-s>`_ and `Nano X <https://shop.ledger.com/products/ledger-nano-x>`_ are hardware wallets built by `Ledger <https://www.ledger.com/>`_.
+
+A hardware wallet is a separate device that stores the private keys for your crypto assets. The keys never leave the device and remain secure even if the device is connected to a compromised computer.
+
+Insolar application for Ledger Nano S and X keeps your Insolar private key on the device and lets you manage XNS coins via the `Insolar Wallet <https://wallet.insolar.io>`_.
+
+The application is developed by Insolar team. To request technical support, send an email to support@insolar.io.
```

Comment:
The application **has been** developed by **the** Insolar team.

Reply:
There's a guideline that documentation should be in the present tense. Plus, the development is not finished (and never will be), so, technically it should be present continuous.

Reply:
Ok, ignore. Still, add the to Insolar team